src/components/dialog/demoBasicUsage/script.js

```diff
@@ -4,24 +4,32 @@ angular.module('dialogDemo1', ['ngMaterial'])
   $scope.alert = '';
 
   $scope.showAlert = function(ev) {
-    $mdDialog.show(
-      $mdDialog.alert()
```

Comment:
@marcysutton - why did you replace the 'API-chaining' approach with the 'configuration object' approach?